src/main/java/com/exadel/sandbox/address/service/AddressServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.exadel.sandbox.address.service;
+
+import com.exadel.sandbox.address.dto.AddressBaseDto;
+import com.exadel.sandbox.address.dto.AddressCreateDto;
+import com.exadel.sandbox.address.dto.AddressResponseDto;
+import com.exadel.sandbox.address.dto.AddressUpdateDto;
+import com.exadel.sandbox.address.entity.Address;
+import com.exadel.sandbox.address.repository.AddressRepository;
+import com.exadel.sandbox.exception.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class AddressServiceImpl implements AddressService {
+
+    private final AddressRepository addressRepository;
+    private final ModelMapper mapper;
+
+    @Override
+    public List<AddressResponseDto> getAddresses() {
```

Comment:
Please rewrite this method(cycle for) something like that:

        return addresses.stream()
                .map(attachment -> mapper.map(addresses, AddressResponseDto.class))
                .collect(Collectors.toList());

And also please check other places and try to use stream where possible